assets/lang/th_TH.ini

```diff
@@ -885,17 +886,15 @@ Free space = พื้นที่ว่าง
 MemoryStickDescription = เลือกแหล่งที่เก็บข้อมูล PSP (เม็มโมรี่ สติ๊ก)
 Move Data = ย้ายข้อมูล
 New PSP Data Folder = โฟลเดอร์เก็บข้อมูล PSP อันใหม่
-No data will be changed. = ไม่มีข้อมูลอะไรถูกเปลี่ยนแปลง
+No data will be changed = ไม่มีข้อมูลอะไรถูกเปลี่ยนแปลง
 Old PSP Data Folder = โฟลเดอร์เก็บข้อมูล PSP อันเก่า
-PPSSPP will restart after the change. = PPSSPP จะรีสตาร์ทใหม่หลังจากปรับเปลี่ยนตั้งค่า
-ScopedStorageWarning = คำเตือน: การรองรับแอนดรอยด์ที่ใช้ Scoped Storage ยังอยู่ในขั้นทดสอบ\nเพราะฉะนั้นอาจจะเขมือบข้อมูลของคุณ
+PPSSPP will restart after the change = PPSSPP จะรีสตาร์ทใหม่หลังจากปรับเปลี่ยนตั้งค่า
 Starting move... = กำลังเริ่มการย้าย...
 That folder doesn't work as a memstick folder. = โฟลเดอร์นั้นไม่สามารถนำมาใช้เป็นโฟลเดอร์แหล่งที่เก็บข้อมูลได้
-That's the folder being used! = อันนั้นคือโฟลเดอร์ที่ปรับใช้งานอยู่แล้ว!
 USBAccessThrough = การเข้าถึงข้อมูล ผ่านใน Android/data/org.ppsspp.ppsspp/files
 USBAccessThroughGold = การเข้าถึงข้อมูล ผ่านใน Android/data/org.ppsspp.ppssppgold/files
-Use App Private Directory = ใช้แหล่งเก็บข้อมูลแอพแบบส่วนบุคคล
-Use PSP folder at root of storage = ใช้โฟลเดอร์ PSP ที่อยู่ต้นทางของแหล่งเก็บข้อมูลในเครื่อง
+Use App Private Data = ใช้แหล่งเก็บข้อมูลแอพแบบส่วนบุคคล
+Use PSP folder at root of storage = Use PSP folder at root of storage
```

Comment:
Any reason this is back in english? I wounder if you had both file opened and swapped english and thai one :)

Reply:
Haha, yes, that seems to have been what I did :(